src/Http/SessionAuthenticator.php

```diff
@@ -22,6 +22,8 @@ public function logIn(Session $session, $token)
         // Backwards compatibility with $userId as parameter
         // Remove in beta 17
         if (! ($token instanceof AccessToken)) {
```

Comment:
Can we add a line that says $userId = $token right above the trigger_error line so that what we're doing on the generate line is a bit clearer?